libs/net/dal/TNOContext.cs

```diff
@@ -175,7 +175,7 @@ public int CommitTransaction()
                     }
                 }
 
-                _logger?.LogError("{metadataName}: {sb}", metadataName, sb);
+                _logger?.LogError("{metadataName} ({when}): {sb}", metadataName, DateTime.Now, sb);
```

Comment:
This isn't necessary.  All logs have their date and time.  In the terminal console they don't show it, but you can use the Kibana console to see date and time instead.

Reply:
Updated, thanks